tools/operator/fabricconfiguration/genesisconfig/config.go

```diff
@@ -0,0 +1,448 @@
+/*
```

Comment:
All of the functions in this file can be removed, we only use the structs, gut it of all functions.

Reply:
removed the file.